pkg/cmd/codespace/ssh.go

```diff
@@ -45,6 +64,10 @@ func newSSHCmd(app *App) *cobra.Command {
 	sshCmd.Flags().StringVarP(&opts.codespace, "codespace", "c", "", "Name of the codespace")
 	sshCmd.Flags().BoolVarP(&opts.debug, "debug", "d", false, "Log debug data to a file")
 	sshCmd.Flags().StringVarP(&opts.debugFile, "debug-file", "", "", "Path of the file log to")
+	sshCmd.Flags().BoolVar(&opts.stdio, "stdio", false, "Proxy sshd connection to stdio")
+	sshCmd.Flags().MarkHidden("stdio")
+
+	sshCmd.AddCommand(newConfigCmd(app))
```

Comment:
Even though this is most strictly correct nesting, for practical and consistency reasons I don't think I'd want to go with `gh cs ssh config`. This is also because the `cs ssh` commands accepts arbitrary arguments, one of which could conceivably be "config".

This is bike-shedding at this point, but how would you feel about `gh cs ssh-config`? BTW, if this PR is not urgent to merge, we could wait until after the holidays to ask more from my team about how they would imagine the final command to be named.

Reply:
Well, they can be disambiguated:
```
$ gh cs ssh -c znull-cli-cli-4q6gr7qfjjp -- config
zsh:1: command not found: config
shell closed: exit status 127

$ gh cs ssh -c znull-cli-cli-4q6gr7qfjjp config
Host cs.znull-cli-cli-4q6gr7qfjjp.trunk
...
```

But I concede that's kind of hair-splitting. I'm fine with `gh cs ssh-config`. My main reason for not going with that to begin with is that it pollutes the toplevel namespace more than I'd like. It's currently fairly focused:

```
$ gh cs
Connect to and manage your codespaces

USAGE
  gh codespace [flags]

CORE COMMANDS
  code:       Open a codespace in Visual Studio Code
  cp:         Copy files between local and remote file systems
  create:     Create a codespace
  delete:     Delete a codespace
  list:       List your codespaces
  logs:       Access codespace logs
  ports:      List ports in a codespace
  ssh:        SSH into a codespace
  stop:       Stop a running codespace
```

Adding `ssh-config` there next to `ssh` seems a bit unsightly. But I'd be happy to go with whatever the consensus is.

Reply:
Good point about disambiguation. Let's decide on final naming at the start of January.